renku/cli/run.py

```diff
@@ -28,32 +28,47 @@
 * input files if linked to existing files in the repository,
 * output files if modified or created while running the command.
 
+Input and output files and folders can only be detected, if they are passed as
+arguments to ``renku run``.
+
 .. note:: If there were uncommitted changes then the command fails.
    See :program:`git status` for details.
 
-Detecting input files
-~~~~~~~~~~~~~~~~~~~~~
+Detecting input files and folders
```

Comment:
maybe we can refer to these collectively as "paths"? So this would be "Detecting input paths"